op-guide/generate-self-signed-certificates.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 生成自签名证书
+category: deployment
+---
+
+# 生成自签名证书
+
+## 概述
+
+本文档提供使用`cfssl`生成自签名证书的示例。
+
+假设实例集群拓扑如下：
+
+| Name  | Host IP     | Services   |
+| ----- | ----------- | ---------- |
+| node1 | 172.16.10.1 | PD1, TiDB1 |
+| node2 | 172.16.10.2 | PD2, TiDB2 |
+| node3 | 172.16.10.3 | PD3        |
+| node4 | 172.16.10.4 | TiKV1      |
+| node5 | 172.16.10.5 | TiKV2      |
+| node6 | 172.16.10.6 | TiKV3      |
+
+
+
+##下载 cfssl
```

Comment:
Leave a space after the "##", to avoid possible display error on the official website page. Please modify all similar items involved in this PR.